cypress/integration/sales/void_a_sales_invoice_then_invoice_the_billing_candidates_again.js

```diff
@@ -0,0 +1,372 @@
+/*
+ * #%L
```

Comment:
what's that ? I never noticed this :)

Reply:
we add this header to new java file; we didn't aggree on an approach for javascript so far..maybe we should.
I think this is coming out of cristi's IDE

Reply:
It's not just my ide, this licence string is a copy-pasta from what existing previously in all the java files w/o me modifying anything except the project name.
Both `#%L` and `%%` were already there.

Maybe it's time we read a bit if we really need each big licence header in all the files, or if it's enough to have either:
- a 1 liner "Licenced to metasfresh, read the licence inside "LICENCE.md" or "LICENCE.txt" files at the root of the project."
- no licence string at all since each repository already has the "LICENCE.md" and "LICENCE.txt" files

Btw: why are there 2 licence files?

Reply:
🤷‍♂ I'll leave the licensing decisions to Tobi and Mark as it's their pet :)